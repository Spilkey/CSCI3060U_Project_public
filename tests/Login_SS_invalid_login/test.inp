login
u___ser01
logout