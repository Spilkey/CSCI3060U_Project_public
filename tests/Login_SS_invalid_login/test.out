00_u___ser04__________SS_000000000